packages/constants/src/applianceEvents.js

```diff
@@ -0,0 +1,6 @@
+export const STARTING = 'starting'
+export const READY = 'ready'
+export const ERROR = 'error'
```

Comment:
Apparently `error` is a [special case](https://nodejs.org/docs/latest-v14.x/api/events.html#events_error_events) for event emitters in Node.

Specifically, if we use this, then if an appliance emits an error and nothing is listening, that will full-blown crash node.

That wasn't my intent when I picked this, though it might be an appropriate behavior.  Just wanted to flag it so we can talk about it as part of this PR